<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_house_form</class>
 <widget class="QWidget" name="add_house_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add House</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>571</width>
     <height>491</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>80</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="id_lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_address1">
       <property name="text">
        <string>Address 1:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="address1_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_address2">
       <property name="text">
        <string>Address 2:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="address2_lineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_city">
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="city_lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_state">
       <property name="text">
        <string>State:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="state_lineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_postal">
       <property name="text">
        <string>Postal Code:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="postalcode_lineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_country">
       <property name="text">
        <string>Country:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="country_lineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_size">
       <property name="text">
        <string>Size (mÂ²):</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="size_lineEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>Date of Register:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDateEdit" name="date_dateEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add_push_button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="photo_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>292</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse..</string>
    </property>
   </widget>
   <widget class="QLabel" name="photo_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>No Photo</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
